lib/config-util.js

```diff
@@ -14,7 +14,7 @@ var Config = {}
 Config.loadConfig = function (argv, cwd) {
   cwd = cwd || process.env.NYC_CWD || process.cwd()
   var pkgPath = findUp.sync('package.json', {cwd: cwd})
-  var rcPath = findUp.sync('.nycrc', {cwd: cwd})
+  var rcPath = findUp.sync(['.nycrc', '.nycrc.json'], {cwd: cwd})
```

Comment:
Ignore me, I was ending up with the wrong version of `find-up` for some reason which broke this. Deleting comments now.